### ISSUE TITLE
Duplicate identifiers in let declaration should be a syntax error

### ISSUE THREAD

**Issue by btmills** [eslint/espree#127]
From [13.2.1.1](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-let-and-const-declarations-static-semantics-early-errors):

> It is a Syntax Error if the BoundNames of _BindingList_ contains any duplicate entries.

``` js
console.log(require("util").inspect(require("./").parse(
    "let foo, foo;",
    { ecmaFeatures: { blockBindings: true } }
), { depth: 99 }));
```

This parses fine, but my understanding is that it should be a syntax error.
##

**Comment by nzakas** [comment]
Looks like this is still broken because Acorn doesn't throw an error here.

**Comment by not-an-aardvark** [comment]
This was fixed in Acorn awhile ago (https://github.com/ternjs/acorn/commit/53dfd4b65ee9721b3537c8d22f12f2592517f177)
